tests/test_index_order.py

```diff
@@ -121,7 +129,14 @@ def test_index_ordering():
         cmd = "_".join(template.split("_")[2:])
         cmd_len = len(cmd)
         check_val, check_msg = check_order(
-            os, prior_os, cmd_len, prior_len, OS_CHOICES, used_os, cmd, prior_cmd
+            os,
+            prior_os,
+            cmd_len,
+            prior_len,
+            OS_CHOICES,
+            used_os,
+            cmd,
+            prior_cmd,
```

Comment:
This is an unnecessary change, please revert.

Reply:
Will do!